.drone.yml

```diff
@@ -33,12 +33,51 @@ steps:
       DIRECTUS_TOKEN:
         from_secret: directus_token
     commands:
+      - export NODE_ENV=staging
+      - npm run export:production
+    when:
+      branch:
+        exclude:
+          - master
+
+  - name: deploy staging
+    image: cupcakearmy/drone-deploy
+    pull: always
+    environment:
+      PLUGIN_KEY:
+        from_secret: key
+    settings:
+      host: love-foundation.org
+      user: root
+      port: 1312
+      target: /srv/staging
```

Comment:
irgendwann können wir vermutlich /srv/web/{prod,staging} machen aber ein ander mal.

Reply:
Yes, makes sense. I guess when we release we can make this adjustment :)